cider-velox/src/ArrowDataConvertor.h

```diff
@@ -33,13 +33,18 @@ class ArrowDataConvertor : public DataConvertor {
  public:
   ArrowDataConvertor() {}
 
+  explicit ArrowDataConvertor(std::shared_ptr<CiderAllocator> allocator);
+
   CiderBatch convertToCider(RowVectorPtr input,
                             int num_rows,
                             std::chrono::microseconds* timer) override;
 
   RowVectorPtr convertToRowVector(const CiderBatch& input,
                                   const CiderTableSchema& schema,
                                   memory::MemoryPool* pool) override;
+
```

Comment:
> `convertToRowVector` should use `allocator` to replace `pool`

Offline discussion draws conclusions that we will use memory::MemoryPool* to manage memory here. Because in  convertToRowVector function，we used memory::MemoryPool* in thirdparty velox. So we do this for consistent memory management